packages/main/src/TimePicker.ts

```diff
@@ -187,6 +187,16 @@ type TimePickerInputEventDetail = TimePickerChangeInputEventDetail;
 		},
 	},
 })
+/**
+ * Fired after the value-help dialog of the component is opened.
+ * @public
+ */
+@event("open")
```

Comment:
since we've made preventable events, you have to use the right JSDoc to mark the event as such `@allowPreventDefault`

Reply:
I think the preventing was a misunderstanding and they should not be preventable, because there are no realistic scenarios where an app would want to not open a timepicker when the user clicks the icon or vice-versa. If they request it one day, we can always add it.